src/components/Chat/AvailableVolunteersComponent.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+interface IProps {
+  availableVolunteers: string[];
+  closePopup(): any;
```

Comment:
Ikke ha explicit any. Heller bruker ordentlige typer. Hvis det er en close funksjon antar jeg at den returnerer void.

Du bruker heller ikke closePopup så ville anbefalt å fjerne alle props fra denne komponenten og bruke useContext for å hente available volounteers